### ISSUE TITLE
[Bug]: startup error with react scripts

### ISSUE THREAD

**Issue by sergiycheck** [wallabyjs/console-ninja#101]
### What happened?

I could not start the react app after uninstalling console.ninja and wallaby and have to manually remove folders

```json
"react-scripts": "4.0.3",
```
node version
```bash
$ node -v
v14.18.1
```
Os version
```bash
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.1 LTS
Release:	22.04
Codename:	jammy

```

### Version

latest

### Steps to reproduce (or sample repo)

Start react app with react-scripts and intall lates console ninja vscode extension (update 08_02_2023)
```json
"react-scripts": "4.0.3",
```

### Log output

```shell
Starting the development server...

/home/serhii/projects/cyberflip_projects/frontend-app/old-branch/web/node_modules/react-scripts/scripts/start.js:19
  throw err;
  ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at new NodeError (internal/errors.js:322:7)
    at validateString (internal/validators.js:124:11)
    at Object.extname (path.js:1358:5)
    at ah._isFileWithSupportedExtension (/home/serhii/.vscode/extensions/wallabyjs.console-ninja-0.0.73/out/buildHook/index.js:1:2139112)
    at /home/serhii/.vscode/extensions/wallabyjs.console-ninja-0.0.73/out/buildHook/index.js:1:2138621
    at Array.filter (<anonymous>)
    at ah.files (/home/serhii/.vscode/extensions/wallabyjs.console-ninja-0.0.73/out/buildHook/index.js:1:2138586)
    at pw.filesToInstrument (/home/serhii/.vscode/extensions/wallabyjs.console-ninja-0.0.73/out/buildHook/index.js:1:3469336)
    at pw.start (/home/serhii/.vscode/extensions/wallabyjs.console-ninja-0.0.73/out/buildHook/index.js:1:3469621)
    at fm.processMessageReceived (/home/serhii/.vscode/extensions/wallabyjs.console-ninja-0.0.73/out/buildHook/index.js:1:2152226) {
  code: 'ERR_INVALID_ARG_TYPE'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by fullstackzach** [comment]
I got this same issue starting today..

```
npm run test-unit 

> api@0.1.0 test-unit
> jest --detectOpenHandles

Debugger listening on ...
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.
Waiting for the debugger to disconnect...
node:internal/errors:464
    ErrorCaptureStackTrace(err);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at new NodeError (node:internal/errors:371:5)
    at validateString (node:internal/validators:120:11)
    at Object.extname (node:path:1386:5)
    at ah._isFileWithSupportedExtension (/Users/.../.vscode/extensions/wallabyjs.console-ninja-0.0.73/out/buildHook/index.js:1:2139112)
    at /Users/.../.vscode/extensions/wallabyjs.console-ninja-0.0.73/out/buildHook/index.js:1:2138621
    at Array.filter (<anonymous>)
    at ah.files (/Users/.../.vscode/extensions/wallabyjs.console-ninja-0.0.73/out/buildHook/index.js:1:2138586)
    at pw.filesToInstrument (/Users/.../.vscode/extensions/wallabyjs.console-ninja-0.0.73/out/buildHook/index.js:1:3469336)
    at pw.start (/Users/.../.vscode/extensions/wallabyjs.console-ninja-0.0.73/out/buildHook/index.js:1:3469621)
    at pw._startIfRequired (/Users/.../.vscode/extensions/wallabyjs.console-ninja-0.0.73/out/buildHook/index.js:1:3468982) {
  code: 'ERR_INVALID_ARG_TYPE'
}
make: *** [test-unit] Error 1
```

the same even happens if a run my tests without my debugger.

**Comment by ArtemGovorov** [comment]
Thanks for the report. The issue is fixed and the fix is published in v0.0.74+.
